### PR TITLE
feat: add material-ui integration, next-seo

### PR DESCRIPTION
**Pull request by abui-am**
add material ui integration with nextjs and design palette
add next-seo for SEO

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -1,17 +1,31 @@
...
+    if (jssStyles) {
+      jssStyles.parentElement.removeChild(jssStyles);
+    }
+  });
+
+  const queryClient = new QueryClient();
 
-const MyApp = ({ Component, pageProps }: AppProps) => {
   return (
     <QueryClientProvider client={queryClient}>
```
Comment (irhamputra) [inline pages/_app.tsx:23]: loh Next-SEO Component nya belom di implement?

Comment (abui-am) [reply pages/_app.tsx:23]: Ahh, okee mas

File: pages/_app.tsx [hunk 2]
```diff
@@ -1,17 +1,31 @@
 import '@/styles/globals.css';
```
Comment (irhamputra) [inline pages/_app.tsx:1]: kalo udah integrasi Material UI nya bisa di delete style folder dan file nya, ga perlu ada lagi @abui-am

**Review by irhamputra — CHANGES_REQUESTED** [review]
tolong, Implement Next-SEO dan delete styled foldet dan filesnya

File: pages/_app.tsx (hunk 2 above)
Comment (abui-am) [reply pages/_app.tsx:1]: mas, kalau ini masuknya commit message apa ya? chore / feat/ refactor ?

Comment (irhamputra) [reply pages/_app.tsx:1]: @abui-am Refactor om

**Review by irhamputra — APPROVED** [review]
Good job @abui-am

**Pull request merged**